shinysdr/plugins/wspr/demodulator.py

```diff
@@ -225,6 +243,15 @@ def filename(self, start_time):
         time_str = time.strftime(b'%y%m%d_%H%M.wav', time.gmtime(start_time))
         filename = b'%s_%s' % (self.context.get_absolute_frequency_cell().get(), time_str)
         return os.path.join(self.directory, filename)
+    
+    @exported_value(type=unicode, label='Status', changes='explicit')
+    def get_status(self):
+        if self.__decoder_active:
+            return 'Decoding…'
+        elif self.__frequency_subscription and not self.__invalidated_by_frequency_change:
+            return 'Receiving'
```

Comment:
This might imply if it's decoding, it's not receiving, which isn't true. If there are a lot of signals, it's not unusual for `wsprd` to still be running at the start of the next interval. Wouldn't want people thinking it wasn't receiving when the next interval started.

I'd say just indicate "decoding" or nothing. Receiving would better be indicated with a progress bar, or maybe a countdown timer since I haven't seen progress bars in the UI yet.